programs/protocol_event/src/state/participant.rs

```diff
@@ -0,0 +1,32 @@
+use crate::state::grouping::{MAX_CODE_LENGTH, MAX_NAME_STRING_LENGTH};
```

Comment:
i know this is not defined here but the name of the constants seems to be not following the same pattern. I'm referring to one containing the word "STRING".